lib/active_graphql/model.rb

```diff
@@ -37,16 +37,19 @@ def initialize(attributes)
         @attributes = attributes.deep_transform_keys { |it| it.to_s.underscore.to_sym }
       end
 
-      def update(params)
-        action_name = "update_#{self.class.active_graphql.resource_name}"
+      def mutation_method(action_name, params = {})
```

Comment:
it does not returns `method` so I would recommend to name it to something like `mutation`

```suggestion
      def mutation(action_name, params = {})
```

Reply:
Perfect name for new born! :rocket: